zmsadmin/js/page/availabilityDay/index.js

```
@@ -441,72 +489,110 @@ class AvailabilityPage extends Component {
                     }
-                    hideSpinner();
                 }
-            )
+            );
+        });
+    }
+    
+    validateAvailabilityList(availabilitylist) {
+        const timeRegex = /^\d{2}:\d{2}(:\d{2})?$/;
+    
+        const isValidTimestamp = (timestamp) => !isNaN(timestamp) && moment.unix(timestamp).isValid();
```

Biome (1.9.4)

[error] isNaN is unsafe. It attempts a type coercion. Use Number.isNaN instead.

See the MDN documentation for more details.
Unsafe fix: Use Number.isNaN instead.


(lint/suspicious/noGlobalIsNan)